cloud-run/src/check-sa.js

```diff
@@ -0,0 +1,37 @@
+const exec = require('@actions/exec');
+
+const getServiceAccounts = async (project) => {
+  let output = '';
+  await exec.exec('gcloud', [
+    'iam',
+    'service-accounts',
+    'list',
+    `--project=${project}`,
+    '--format=value(EMAIL)',
+  ], {
+    silent: true,
+    listeners: {
+      stdout: (data) => {
+        output += data.toString('utf8');
+      },
+    },
+  });
+  return output.split(/[\r\n]+/);
+};
+
+const checkServiceAccount = async (serviceName, projectID) => {
+  let found = false;
+
+  const emails = await getServiceAccounts(projectID);
+  for await (const email of emails) {
```

Comment:
I would write this using Array.find instead.
```
const match = emails.find((email) => email === `${serviceName}@${projectID}.iam.gserviceaccount.com`);
return !!match
```

Reply:
yeah that looks great :D